### PR TITLE
Add support for missing typings

### PR DESCRIPTION
**Pull request by ScreamZ**
Add support fort missing typings

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## johnny-five ([<kbd>unpkg</kbd>](https://unpkg.com/browse/johnny-five@latest/))
was missing the following properties:
1. Color
1. Expander
1. GPS
1. SIP
1. Keypad

<details>
<summary>as well as these 13 other properties...</summary>
<p>LedControl, Light, Repl, Multi, Luxmeter, Magnetometer, Buttons, Leds, Pins, Sensors, Servos, Switches, Touchpad</p>
</details>


## Formatting

The following files are not formatted:
1. types/johnny-five/index.d.ts

Consider running `npx dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a0fa4026f81ba85fd95353d848107a537f64cfee
</p>

File: types/johnny-five/johnny-five-tests.ts
Comment (MysteryBlokHed) [inline types/johnny-five/johnny-five-tests.ts:1]: A lot of changes have been added here; it'd probably be worth adding new tests for the new functionality

Comment (ScreamZ) [reply types/johnny-five/johnny-five-tests.ts:1]: I just did it :)

**Review by scottgonzalez — APPROVED** [review]

**Comment by ScreamZ** [comment]
Ready to merge

**Pull request merged**